api/logs/permissions.py

```diff
@@ -12,8 +12,14 @@ class ContributorOrPublicForLogs(permissions.BasePermission):
     def has_object_permission(self, request, view, obj):
         assert isinstance(obj, (NodeLog)), 'obj must be a NodeLog, got {}'.format(obj)
 
-        for node_id in obj._backrefs['logged']['node']['logs']:
-            node = get_object_or_error(Node, node_id, display_name='node')
-            if ContributorOrPublic().has_object_permission(request, view, node):
+        if obj._backrefs.get('logged'):
```

Comment:
Unless @caneruguz has told you differently, I was told to hold off on event tracking for a few days(as the dashboard itself is changing very rapidly right now).  Caner, @hmoco, and I are responding to Steve's review right now.  When the dashboard is more settled, then I will bring this up to date.  I think the review should wait until then.